### PR TITLE
add [entertainment system] session

### PR DESCRIPTION
**Pull request by gasgas4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +109,10 @@ the table of content before making pull requests.
 - [Hackers Remotely Attack a Jeep on the Highway](http://www.wired.com/2015/07/hackers-remotely-kill-jeep-highway/)
 - [Comprehensive Experimental Analyses of Automotive Attack Surfaces](http://static.usenix.org/events/sec11/tech/full_papers/Checkoway.pdf)
 
+### entertainment system
```
Comment (0xbadshah) [inline README.md:112]: Would be better if you capitalise the letter 'e'.

Comment (nebgnahz) [reply README.md:112]: Maybe also `S`.

**Comment by nebgnahz** [comment]
Thanks for the PR!

I am wondering if it makes more sense to categorize this as `Flight` or `Airplanes` rather than `Entertainment System`. The flight is the hacked target while the entertainment system is the vulnerability. This is similar to the Jeep case, in which the Jeep is the target while `Uconnect` is the vulnerability.

Thought?

**Comment by gasgas4** [comment]
Planes category is sense to me.   Let me close this and make this one.

**Pull request closed without merging**